webapp/create_aois.py

```diff
@@ -12,14 +13,30 @@ def exec_sql(sql):
         with connection.cursor() as cursor:
             cursor.execute(sql)
 
-parser = argparse.ArgumentParser()
-parser.add_argument('--output_geojson', required=True)
-parser.add_argument('--hull_algorithm', choices=['concave', 'convex'], default='convex')
-parser.add_argument('--with_network_centrality', action='store_true', default=False)
+
+def is_valid_file(parser, arg):
+    try:
+        boundary_4326 = gpd.read_file(arg)
+        boundary_4326 = boundary_4326.to_crs({'init': 'epsg:4326'})
+        return boundary_4326
+    except (Exception) as e:
+        raise argparse.ArgumentTypeError(e) from e
+
+
+
+parser = argparse.ArgumentParser(description='Used to export the AOIs extract`')
+parser.add_argument('DEST', help='file path for the exported AOIs')
```

Comment:
Please use `'dest', metavar='DEST'` if you want it to be displayed in ALL CAPS. Let's keep Python variables lowercase. See [metavar](https://docs.python.org/3/library/argparse.html#metavar) in the argparse documentation.

(Arguably, argument values are kinda constant in that you shouldn't change them during runtime, but it's not customary to have that reflected in their variable names.)